website/static/js/comment.js

```diff
@@ -147,19 +147,20 @@ BaseComment.prototype.fetchNext = function(url, comments) {
         {'isCors': true});
     request.done(function(response) {
         comments = comments.concat(response.data);
-        if (response.links.next !== null) {
-            self.fetchNext(response.links.next, comments);
-        } else {
-            self.comments(
-                ko.utils.arrayMap(comments, function(comment) {
-                    return new CommentModel(comment, self, self.$root);
-                })
-            );
-            deferred.resolve(self.comments());
-            self.configureCommentsVisibility();
+        if (self._loaded !== true) {
             self._loaded = true;
             self.loadingComments(false);
         }
+        self.comments(
+            ko.utils.arrayMap(comments, function(comment) {
```

Comment:
I wonder if the "jumpiness" is due to the fact that we are serializing *all* comments every time a new page of comments is received. Instead, you should only need to serialize comments as they come.